.github/dependabot.yml

```
@@ -38,3 +71,18 @@ updates:
     - kind/enhancement
     - release-note/misc
     - priority/release-blocker
+  - package-ecosystem: docker
+    directory: /
+    schedule:
+      interval: daily
+    commit-message:
+      prefix: "dockerfile:"
+    open-pull-requests-limit: 5
+    rebase-strategy: disabled
+    target-branch: "v0.10"
+    # Only bump the alpine patch version in stable branches
+    update-types: ["version-update:semver-patch"]
```

[CI is unhappy](https://github.com/cilium/hubble/pull/849/checks?check_run_id=10611332689):

```
The property '#/updates/5/' contains additional properties ["update-types"] outside of the schema when none are allowed
```

According to [the dependabot configuration doc](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore) `update-types` is only supported in the `ignore:` section, so to achieve the goal of only bumping patch versions I think this should be:

```suggestion
    # Only bump the alpine patch version in stable branches
    ignore:
        update-types: ["version-update:semver-major", "version-update:semver-minor"]
```